src/main-oe/sgxlkl_run_oe.c

```diff
@@ -327,39 +253,34 @@ static void help_config()
         "  SGXLKL_HD_KEY",
         "Encryption key as passphrase or file path to a key file for the root "
         "file system image (Debug only).\n");
+
+#ifndef SGXLKL_RELEASE
+    printf("\n");
+    printf("Variables to override enclave/app settings [DEBUG/NON-RELEASE "
+           "build only]\n");
+    printf("==================================================================="
+           "======\n");
+
     printf(
```

Comment:
Yeah, `SGXLKL_WG_PEERS` didn't fit into any of the other buckets. In debug mode it overrides all of the peers' settings, not just one of them. I think we could just drop this variable, I can't imagine anyone ever using this variable to override the config file.

Reply:
@prp Should we remove it? It's not used in samples/test.

Reply:
Actually, for testing this could be quite nice, as you can set WG peers from the environment at launch time, without having to edit/rewrite the enclave config. Since it's implemented, I would keep it.

Reply:
You say that about every environment variable, yet I still want to get rid of all of them!

I don't think anybody will need to change the WG peers many times to many different values. And even then, you can do it along the lines of `sed -i 's/"verbose": true/"verbose": false/' enclave-config.json && sgxlkl-run-oe ...`. If you want to get really fancy, you can even stuff the whole JSON into an env var of your choice! E.g. `echo $MYCONFIG > config.json && sgxlkl-run-oe ...`